pkg/queue/breaker_test.go

```diff
@@ -38,8 +38,8 @@ func TestBreakerInvalidConstructor(t *testing.T) {
 		name    string
 		options BreakerParams
 	}{{
-		"QueueDepth = 0",
-		BreakerParams{QueueDepth: 0, MaxConcurrency: 1, InitialCapacity: 1},
+		name:    "QueueDepth = 0",
```

Comment:
if you went down this path, you need to finish it :)
But I think this does not belong to this pr.

Reply:
@vagababov I am not done yet :)